README.md

```diff
@@ -7,34 +7,30 @@ Rails
- 
- - **Minecart**
- Minecart is a rail transport vehicle that can be used by the player to travel far distances faster. You can ride the minecart by pressing `e` while facing the minecart. Although minecart can move without a locomotive by pressing `w` to move forward and pressing `s` to stop, using locomotive is better because it gives a significant boost in speed. Minecart will change direction if it hits an obstacle (Ex: Unlinked Locomotive, Block). You can't push a minecart, it will stay stationary unless you ride on it or a moving locomotive is linked to it. You can destroy minecart by left-clicking while facing it.
- 
- - **Wrench**
- Wrench is a tool that can be used to connect minecarts with a locomotive and also change the direction of a tee joint.
+## Rails
+Rails are the most fundamental part of this module. They are the tracks on which minecarts can travel.
+  
+## Carts
+A cart is a vehicle that can travel on rails. This module contains a simplistic cart which can be ridden by the player. It moves only when pushed by the player or dragged by a connected cart.
+
+## Wrench
+The wrench is a tool that can be used to perform specific actions like connecting carts with each other or changing the direction of a tee joint.
+
+For additional items like locomotives and special rails to use with **Rails**, please take a look at the [AdditionalRails](https://github.com/Terasology/AdditionalRails) module.
 
 Creating Tracks
 =====
-Tracks are build using rails and they are where your minecarts and locomotives will travel on. Tracks can be straight or sloped and can have a tee or a corner joint.
+Tracks are built by placing rail blocks. When a rail block is placed, it automatically connects to nearby rail blocks to form contiguous track segments. Rail blocks can be straight or sloped and can form a corner, T (tee) junction, or an intersection.
 
-Linking Locomotive-Minecart 
+Connecting Carts
 =====
-TODO: still needs to be implemented
 
-You can link a locomotive with a minecart by following this steps:
+You can connect two carts as follows:
 
- 1. Place the locomotive
- 2. Place a minecart behind the locomotive
- 3. Equip your spanner and right-click on the locomotive
- 4. Right click on the minecart if the cart turns green when you hover over it. If the minecart doesn't turn green, try doing it again from step (1) but make sure the locomotive and the minecart is close enough.
- 5. You now have a linkage, between your locomotive and your minecart. You can now turn on the locomotive by pressing `e` while looking at it and the minecart will follow the locomotive.
+ 1. Place two carts close to each other
+ 2. Equip your spanner and right-click on a cart
```

Comment:
Unless there is good reason, I would recommend replacing the word `spanner` with the word `wrench` since not everyone might realize they mean the same thing.